packages/frontend/src/components/SavedQueries/CreateSavedQueryModal.tsx

```diff
@@ -53,7 +67,7 @@ const CreateSavedQueryModal: FC<CreateSavedQueryModalProps> = ({
                             text="Save"
                             type="submit"
                             onClick={(e) => {
-                                mutate({ ...savedData, name });
+                                mutate({ ...savedData, name, description });
```

Comment:
default description is '' . Does this mean we will always   (if no description is provided) set description to `empty string` in DB? rather than NULL / undefined ...